content/scaling-kubernetes-cluster/scale-cluster-horizontally.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Scale Kubernetes cluster horizontally"
+description: "Procedures to scale Kubernetes cluster horizontally by adding nodes"
+weight: 1
+---
+
+{{% dialog type="warning" icon="fa-info-circle" title="Prerequisites" %}}
+This assumes you've followed the [Geodesic Module Usage]({{< relref "geodesic/module/with-kops.md" >}}) guide which covers all the scaffolding necessary to get started.
+{{% /dialog %}}
+
+Kops cluster can be scaled horizontally by adding EC2 instances. 
+
+Kops supports rolling cluster upgrades where the master and worker nodes are upgraded one by one.
+
+First, in the module's `Dockerfile`, increase `NODE_MAX_SIZE` and `NODE_MIN_SIZE` to reflect the desired number of worker nodes
+
+{{% dialog type="code-block" icon="fa fa-code" title="Dockerfile Kops config" %}}
+```dockerfile
+ENV BASTION_MACHINE_TYPE="t2.medium"
+ENV MASTER_MACHINE_TYPE="t2.large"
+ENV NODE_MACHINE_TYPE="t2.large"
+ENV NODE_MAX_SIZE="3"
+ENV NODE_MIN_SIZE="3"
+```
+{{% /dialog %}}
+
+Rebuild the `geodesic` shell by running
+
+```sh
+make docker/build
+```
+
+Then, ensure that the `kubectl` context has been set. In `geodesic` shell run
+
+```sh
+kops export kubecfg $KOPS_CLUSTER_NAME
+```
+
+Check and apply the latest Kubernetes update
+
+```sh
+kops upgrade cluster --yes
+```
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+The `--yes` option immediately applies the changes. Not specifying the `--yes` option shows only the changes that are applied.
```

Comment:
Not specifying `--yes` works like `terraform plan` and shows the pending changes, no?